t/16-utils-runcmd.t

```diff
@@ -95,13 +95,14 @@ subtest 'git commands with mocked run_cmd_with_log_return_error' => sub {
     # configure update branch and remote
     $git->config->{update_remote} = 'origin';
     $git->config->{update_branch} = 'origin/master';
+    $git->config->{do_cleanup} = 'yes';
     is($git_config->{update_remote}, $git->config->{update_remote}, 'global git config reflects all changes');
 
     # test set_to_latest_master (non-error case)
     is($git->set_to_latest_master, undef, 'no error');
     is_deeply(
         \@executed_commands,
-        [[qw(git -C foo/bar remote update origin)], [qw(git -C foo/bar rebase origin/master)],],
+        [[qw(git -C foo/bar remote update origin)], [qw(git -C foo/bar rebase origin/master)], [qw(git -C foo/bar reset --hard HEAD)],],
```

Comment:
Shouldn't the reset happen before the rebase?

Reply:
Ah yes, I didn't read it carefully enough